yaml/hints.nim

```diff
@@ -111,148 +110,121 @@ template advanceTypeHint(ch: char) {.dirty.} =
   typeHintStateMachine ch:
   of '~': ythInitial => ythNULL
   of '.':
-    [yth0, ythInt1Zero, ythInt1, ythInt2, ythInt3, ythInt4, ythInt] => ythDecimal
-    [ythInitial, ythMinus] => ythPoint
-    ythSecond2             => ythFraction
+    [ythInt1, ythInt2, ythInt3, ythInt4, ythInt] => ythDecimal
+    [ythInitial, ythMinus, ythPlus] => ythPoint
+    ythSecond2 => ythFraction
   of '+':
+    ythInitial => ythPlus
     ythNumE => ythNumEPlusMinus
     [ythFraction, ythSecond2] => ythAfterTimePlusMinus
   of '-':
-    ythInitial                => ythMinus
-    ythNumE                   => ythNumEPlusMinus
-    [ythInt4, ythInt4Zero]    => ythYearMinus
-    ythMonth1                 => ythMonthMinusNoYmd
-    ythMonth2                 => ythMonthMinus
+    ythInitial => ythMinus
```

Comment:
Why was the alignment removed? Are you using a code formatter? While I generally like using formatters to prevent styling debates, this project hasn't configured one so I'd like to keep the style as-is for the time being.

Reply:
Good question, this was done by accident, I will revert the formatting.